src/v2/Apps/Feature/Components/FeatureHeader/FeatureHeaderDefault.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components"
 import { Flex, HTML, Join, Spacer, Text, color } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FeatureHeaderDefault_feature } from "v2/__generated__/FeatureHeaderDefault_feature.graphql"
-import { NAV_BAR_HEIGHT } from "v2/Components/NavBar"
+import { DESKTOP_NAV_BAR_HEIGHT } from "v2/Components/NavBar"
```

Comment:
Is good to rename this since I think the `NAV_BAR_HEIGHT` was sometimes imported thinking it's only one height ever. I added a note in the constants file to just use the hook. Though we can eventually get rid of this entirely.